src/gpu/intel/ocl/reusable_softmax.hpp

```diff
@@ -159,6 +163,7 @@ struct reusable_softmax_fwd_t : public gpu_primitive_t {
             conf.is_logsoftmax = is_logsoftmax();
             conf.src_data_type = src_dt;
             conf.dst_data_type = dst_dt;
+            conf.subgroup_size = mayiuse_sg(16, compute_engine) ? 16 : 8;
```

Comment:
Shouldn't you be testing if 8 is an acceptable subgroup size as well. I am not sure if we will support this in future hardware.

Reply:
Switched to 32/16. What you think? Seems like this should be ok.